File: Algoritmo(C)/Alg_S6_9.c
#include <stdio.h>
int main(){
    //variables
    float indice;
    //input
    printf("Digite o indice de poluicao: ");
    scanf("%f",&indice);
    //process + output
    if (indice >= 0.5){
        printf("As atividades de todos os grupos estao suspensas");
    }else if (indice >= 0.4){
        printf("As atividades dos grupos 1 e 2 estao suspensas");
    }else if (indice >= 0.3){
        printf("As atividades do grupo 1 estao suspensas");
    }else {
        printf("Nenhum grupo esta suspenso");
    }
}

File: Algoritmo(Python)/Alg_S6_9.py
#input
poluicao = float(input("Digite o valor da poluição: "))
#process
if(poluicao >= 0.5):
    print("Todos os grupos: Suspender Atividades")
elif(poluicao >= 0.4):
    print("Grupos 1 e 2: Suspender Atividades")
elif(poluicao >= 0.3):
    print("Grupo 1: Suspender Atividades")
else:
    print("Níveis Aceitáveis")